refresh_discussion_count.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
Instead a standalone Python script, I suggest implementing a Django management command, following many of the existing commands in the repository.

Reply:
You're right, this file is meant to be removed - I have another management command